### DISCUSSION TITLE
Adding iPhone Support to IINA

### DISCUSSION THREAD

**Discussion by aumchauhan** [iina/iina#4643] [Ideas]
Any plans to bring IINA to iPhone? It's an awesome macOS media player and having it in iOS would be fantastic...

**Comment by low-batt** [comment]
The answer provided in the FAQ question [Will there be a Windows/Linux/iOS version?](https://github.com/iina/iina/wiki/FAQ#will-there-be-a-windowslinuxios-version) is…
> NO

The reason is that IINA is a full GUI for the [mpv](https://github.com/mpv-player/mpv) player specifically designed for macOS. Supporting other platforms would require so many changes that it is deemed better to implement that as a completely separate project.

The mpv wiki contains a list of [GUI frontends](https://github.com/mpv-player/mpv/wiki/Applications-using-mpv#gui-frontends). That page lists:
> [Outplayer](https://apps.apple.com/us/app/outplayer/id1449923287) (iOS)

The author of Outplayer mentions being inspired by IINA.
